src/main/java/com/uipath/uipathpackage/UiPathAssets.java

```diff
@@ -103,13 +102,22 @@ public void perform(@Nonnull Run<?, ?> run,
         util.validateRuntime(launcher);
 
         try {
-            EnvVars envVars = run.getEnvironment(listener);
+            EnvVars envVars = TaskScopedEnvVarsManager.selectOnlyRequiredEnvironmentVariables(run, env, listener);
+
+            CliDetails cliDetails = util.getCliDetails(run, listener, envVars, launcher);
+            String buildTag = envVars.get("BUILD_TAG");

```

Comment:
This variable is present for all types of projects like freestyle, and multibranch pipeline. The reason for asking is because a similar WORKSPACE variable is a part freestyle type of project but not of multibranch pipeline type of project.

Reply:
Yes, Build_Tag works on Pipeline, Free style and multibranch projects.